person/src/com/person/utils/Input.java

```diff
@@ -0,0 +1,35 @@
+package com.person.utils;
+
+import java.util.Scanner;
+
+public class Input {
+    private static Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Reads number from input.
+     * @param prompt
+     * @return
+     */
+    public static int number(String prompt)
+    {
+        System.out.print(prompt + ": ");
+
+        return scanner.nextInt();
+    }
+
+    /**
+     * Reads string from input.
+     * @param prompt
+     * @return
+     */
+    public static String string(String prompt)
```

Comment:
@tatevikb what mean string , please make method meaningfully also comment is not descriptive as well